Content.Shared/Cargo/CargoOrderData.cs

```diff
@@ -6,7 +6,9 @@ namespace Content.Shared.Cargo
     [NetSerializable, Serializable]
     public sealed class CargoOrderData
     {
-        public int OrderNumber;
+        public int OrderIndex;
+        /// The human-readable number, when displaying this order
+        public int OrderNumber { get { return OrderIndex + 1; } }
```

Comment:
Just make a new property rather than changing the existing one as it's more subject to issues.

Reply:
So, in changeset [e9d6cf3](https://github.com/space-wizards/space-station-14/pull/12618/commits/e9d6cf36511da3a07dad150da49c6db48f329984), I renamed OrderNumber to OrderIndex. The game builds and runs properly with that change and I think it improves the code quality, as the name now reflects the usage.

Then, in the following changeset, I added an OrderNumber property, since there were a couple (2) of locations that wanted a number to print on the manifest. 

Because I did applied the changes in that order, I'm confident that there's no usages of "OrderNumber" where the code is really expecting an "OrderIndex" Are you suggesting that the "new" OrderNumber should be renamed to something different entirely? PrintedOrderNumber, maybe?